packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -954,7 +954,7 @@ describe('preset', () => {
         return '/node_modules/react-native-js-preset/jest-preset.js';
       }
 
-      if (name === 'doesnt-exist') {
+      if (/doesnt-exist/.test(name)) {
```

Comment:
this test would pass before as well. Could you either tweak this or add a new test that asserts we throw the correct error?

Reply:
There is a test that checks for the correct exception ("throws when preset not found"), but it was subtly broken. This is the tweak that makes the test fail without my changes.

The short explanation is that `Resolver.findNodeModule` is called with (and should return null for) `"doesnt-exist/jest-preset"` and `"doesnt-exist"`, but this mock only covered the second case.

Reply:
ah, this inside a mock thing 👍 

```suggestion
      if (name.includes('doesnt-exist')) {
```